components/fontloader/fontloader.cpp

```diff
@@ -340,51 +340,48 @@ namespace Gui
             code->addAttribute("size", MyGUI::IntSize(static_cast<int>(data[i].width), static_cast<int>(data[i].height)));
 
             // More hacks! The french game uses several win1252 characters that are not included
-            // in the cp437 encoding of the font. Fall back to similar available characters.
-            if (mEncoding == ToUTF8::CP437)
+            // in the font. Fall back to similar available characters.
+            std::multimap<int, int> additional; // <cp437, unicode>
+            additional.insert(std::make_pair(39, 0x2019)); // apostrophe
+            additional.insert(std::make_pair(45, 0x2013)); // dash
+            additional.insert(std::make_pair(45, 0x2014)); // dash
+            additional.insert(std::make_pair(34, 0x201D)); // right double quotation mark
+            additional.insert(std::make_pair(34, 0x201C)); // left double quotation mark
+            additional.insert(std::make_pair(44, 0x201A));
+            additional.insert(std::make_pair(44, 0x201E));
+            additional.insert(std::make_pair(43, 0x2020));
+            additional.insert(std::make_pair(43, 0x2021));
+            additional.insert(std::make_pair(94, 0x02C6));
+            additional.insert(std::make_pair(37, 0x2030));
+            additional.insert(std::make_pair(83, 0x0160));
+            additional.insert(std::make_pair(60, 0x2039));
+            additional.insert(std::make_pair(79, 0x0152));
+            additional.insert(std::make_pair(90, 0x017D));
+            additional.insert(std::make_pair(39, 0x2019));
```

Comment:
We already have the 2019 above. IMO, there should be the 2018 (LEFT SINGLE QUOTATION MARK).